src/converter/r2t/WrapDispQuoteContent.js

```diff
@@ -4,14 +4,26 @@ export default class WrapDispQuoteContent {
 
   import(dom) {
     dom.findAll('disp-quote').forEach((dispQuote) => {
-      let attribs = dispQuote.findAll('attrib')
+      let attrib = dispQuote.find('attrib')
+      if (!attrib) {
+        // attrib element is required in TextureJATS.
+        attrib = dom.createElement('attrib')
+      }
       let content = dom.createElement('disp-quote-content').append(dispQuote.children)
```

Comment:
Excellent question. We should actually implement that differently. The current implementation works because an element can only have one parent so when:

`dispQuote.append(attrib)` is executed the attrib element gets removed from `disp-quote-content.children`.